index.html

```diff
@@ -23,12 +23,10 @@
     -->
     <script  src="js/index.js"></script>
     <div class="container">
-      <header class="header">
         <h1 id="title" class="text-center">JoeDanSurvey Form</h1>
         <p id="description" class="description text-center">
           Thank you for taking the time to help us improve the platform
         </p>
-      </header>
```

Comment:
I think you should add semantic (header) to your file for proper arrangement of elements and clean code/ readability